packages/components/src/components/icon/brand/ic-brand-derivez.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 52 52"><path fill="#E12E3A" d="M9 0a9 9 0 0 0-9 9v.261L44.871 2.18A8.965 8.965 0 0 0 39 0H9Zm36.797 3.102-8.231 44.897h1.433a9 9 0 0 0 9-9V9c0-2.255-.83-4.318-2.202-5.897Z"/><path fill="#FF444F" fill-rule="evenodd" d="M37.566 48H9a9 9 0 0 1-9-9V9.261L44.871 2.18c.33.285.64.593.926.923L37.566 48Z" clip-rule="evenodd"/><path fill="#fff" d="M12.794 31.276V16.724h9.828v2.75H16.07v2.857h5.817v2.688H16.07v3.507h7.035v2.75h-10.31Zm22.18-12.285c-.448.462-.98 1.064-1.596 1.806a57.32 57.32 0 0 0-1.91 2.395 83.177 83.177 0 0 0-1.954 2.688 46.408 46.408 0 0 0-1.722 2.646h7.413v2.75H23.76v-1.953c.392-.713.868-1.505 1.428-2.372a97.535 97.535 0 0 1 1.785-2.646 64.693 64.693 0 0 1 1.87-2.584 75.06 75.06 0 0 1 1.784-2.247h-6.53v-2.75h10.877v2.267Z"/></svg>
```

Comment:
Get your icon codes from the figma core.
The base size for our viewBox is "0 0 32 32".